diff --git a/node_modules/expo-font/build/ExpoFontLoader.web.js b/node_modules/expo-font/build/ExpoFontLoader.web.js
index 3926897..ce426fb 100644
--- a/node_modules/expo-font/build/ExpoFontLoader.web.js
+++ b/node_modules/expo-font/build/ExpoFontLoader.web.js
@@ -139,7 +139,7 @@ const ExpoFontLoader = {
         return new FontObserver(fontFamilyName, {
             // @ts-expect-error: TODO(@kitten): Typings indicate that the polyfill may not support this?
             display: resource.display,
-        }).load(null, 6000);
+        }).load(null, 60000);
     },
 };
 const isServer = process.env.EXPO_OS === 'web' && typeof window === 'undefined';
diff --git a/node_modules/expo-font/src/ExpoFontLoader.web.ts b/node_modules/expo-font/src/ExpoFontLoader.web.ts
index 8d53d9f..99b599c 100644
--- a/node_modules/expo-font/src/ExpoFontLoader.web.ts
+++ b/node_modules/expo-font/src/ExpoFontLoader.web.ts
@@ -183,7 +183,7 @@ const ExpoFontLoader: Required<ExpoFontLoaderModule> = {
     return new FontObserver(fontFamilyName, {
       // @ts-expect-error: TODO(@kitten): Typings indicate that the polyfill may not support this?
       display: resource.display,
-    }).load(null, 6000);
+    }).load(null, 60000);
   },
 };
 
