diff --git a/node_modules/expo-av/android/src/main/java/expo/modules/av/player/SimpleExoPlayerData.java b/node_modules/expo-av/android/src/main/java/expo/modules/av/player/SimpleExoPlayerData.java
index 8835a39..342667d 100644
--- a/node_modules/expo-av/android/src/main/java/expo/modules/av/player/SimpleExoPlayerData.java
+++ b/node_modules/expo-av/android/src/main/java/expo/modules/av/player/SimpleExoPlayerData.java
@@ -13,6 +13,8 @@ import android.view.Surface;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
+import com.facebook.react.bridge.UiThreadUtil;
+
 import com.google.android.exoplayer2.C;
 import com.google.android.exoplayer2.MediaItem;
 import com.google.android.exoplayer2.PlaybackException;
@@ -120,7 +122,12 @@ class SimpleExoPlayerData extends PlayerData
   public synchronized void release() {
     super.release();
     stopUpdatingProgressIfNecessary();
-    if (mSimpleExoPlayer != null) {
+    if (!UiThreadUtil.isOnUiThread() && mSimpleExoPlayer != null) {
+      UiThreadUtil.runOnUiThread(() -> {
+        mSimpleExoPlayer.release();
+        mSimpleExoPlayer = null;
+      });
+    } else if (mSimpleExoPlayer != null) {
       mSimpleExoPlayer.release();
       mSimpleExoPlayer = null;
     }
