diff --git a/node_modules/expo-font/build/ExpoFontLoader.web.js b/node_modules/expo-font/build/ExpoFontLoader.web.js
index 8230cd1..2914861 100644
--- a/node_modules/expo-font/build/ExpoFontLoader.web.js
+++ b/node_modules/expo-font/build/ExpoFontLoader.web.js
@@ -124,7 +124,7 @@ export default {
         if (!isFontLoadingListenerSupported()) {
             return Promise.resolve();
         }
-        return new FontObserver(fontFamilyName, { display: resource.display }).load(null, 6000);
+        return new FontObserver(fontFamilyName, { display: resource.display }).load(null, 60000);
     },
 };
 const ID = 'expo-generated-fonts';
diff --git a/node_modules/expo-font/src/ExpoFontLoader.web.ts b/node_modules/expo-font/src/ExpoFontLoader.web.ts
index d4e08d9..2d053d3 100644
--- a/node_modules/expo-font/src/ExpoFontLoader.web.ts
+++ b/node_modules/expo-font/src/ExpoFontLoader.web.ts
@@ -163,7 +163,7 @@ export default {
       return Promise.resolve();
     }
 
-    return new FontObserver(fontFamilyName, { display: resource.display }).load(null, 6000);
+    return new FontObserver(fontFamilyName, { display: resource.display }).load(null, 60000);
   },
 };
 
