diff --git a/node_modules/expo-font/build/ExpoFontLoader.web.js b/node_modules/expo-font/build/ExpoFontLoader.web.js
index ef2f136..44dadb2 100644
--- a/node_modules/expo-font/build/ExpoFontLoader.web.js
+++ b/node_modules/expo-font/build/ExpoFontLoader.web.js
@@ -127,7 +127,7 @@ export default {
         if (!isFontLoadingListenerSupported()) {
             return Promise.resolve();
         }
-        return new FontObserver(fontFamilyName, { display: resource.display }).load(null, 6000);
+        return new FontObserver(fontFamilyName, { display: resource.display }).load(null, 60000);
     },
 };
 const ID = 'expo-generated-fonts';
diff --git a/node_modules/expo-font/src/ExpoFontLoader.web.ts b/node_modules/expo-font/src/ExpoFontLoader.web.ts
index 361b15a..6fb09d2 100644
--- a/node_modules/expo-font/src/ExpoFontLoader.web.ts
+++ b/node_modules/expo-font/src/ExpoFontLoader.web.ts
@@ -167,7 +167,7 @@ export default {
       return Promise.resolve();
     }
 
-    return new FontObserver(fontFamilyName, { display: resource.display }).load(null, 6000);
+    return new FontObserver(fontFamilyName, { display: resource.display }).load(null, 60000);
   },
 };
 
