diff --git a/node_modules/expo-font/build/.ExpoFontLoader.web.js.swp b/node_modules/expo-font/build/.ExpoFontLoader.web.js.swp
new file mode 100644
index 0000000..cb4ca40
Binary files /dev/null and b/node_modules/expo-font/build/.ExpoFontLoader.web.js.swp differ
diff --git a/node_modules/expo-font/build/ExpoFontLoader.web.js b/node_modules/expo-font/build/ExpoFontLoader.web.js
index 14752a3..c526aa2 100644
--- a/node_modules/expo-font/build/ExpoFontLoader.web.js
+++ b/node_modules/expo-font/build/ExpoFontLoader.web.js
@@ -139,7 +139,7 @@ const ExpoFontLoader = {
         return new FontObserver(fontFamilyName, {
             // @ts-expect-error: TODO(@kitten): Typings indicate that the polyfill may not support this?
             display: resource.display,
-        }).load(resource.testString ?? null, 12000);
+        }).load(resource.testString ?? null, 60000);
     },
 };
 const isServer = process.env.EXPO_OS === 'web' && typeof window === 'undefined';
diff --git a/node_modules/expo-font/src/ExpoFontLoader.web.ts b/node_modules/expo-font/src/ExpoFontLoader.web.ts
index 98a6637..5338e83 100644
--- a/node_modules/expo-font/src/ExpoFontLoader.web.ts
+++ b/node_modules/expo-font/src/ExpoFontLoader.web.ts
@@ -183,7 +183,7 @@ const ExpoFontLoader: Required<ExpoFontLoaderModule> = {
     return new FontObserver(fontFamilyName, {
       // @ts-expect-error: TODO(@kitten): Typings indicate that the polyfill may not support this?
       display: resource.display,
-    }).load(resource.testString ?? null, 12000);
+    }).load(resource.testString ?? null, 60000);
   },
 };
 
