diff --git a/android/app/build.gradle b/android/app/build.gradle
index 9e0b8bb..b410909 100644
--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -98,6 +98,14 @@ android {
             keyAlias 'androiddebugkey'
             keyPassword 'android'
         }
+        release {
+            if (project.hasProperty('DUOLICIOUS_UPLOAD_STORE_FILE')) {
+                storeFile file(DUOLICIOUS_UPLOAD_STORE_FILE)
+                storePassword DUOLICIOUS_UPLOAD_STORE_PASSWORD
+                keyAlias DUOLICIOUS_UPLOAD_KEY_ALIAS
+                keyPassword DUOLICIOUS_UPLOAD_KEY_PASSWORD
+            }
+        }
     }
     buildTypes {
         debug {
@@ -106,7 +114,7 @@ android {
         release {
             // Caution! In production, you need to generate your own keystore file.
             // see https://reactnative.dev/docs/signed-apk-android.
-            signingConfig signingConfigs.debug
+            signingConfig signingConfigs.release
             shrinkResources (findProperty('android.enableShrinkResourcesInReleaseBuilds')?.toBoolean() ?: false)
             minifyEnabled enableProguardInReleaseBuilds
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
